app/views/batches/show.xml.builder

```diff
@@ -12,7 +12,14 @@ xml.batch do
           request: [
             {
               target_asset: {
-                spiked_in_buffer: [:index, { aliquots: %i[library tag tag2 aliquot_index sample] }],
+                labware: {
+                  spiked_in_buffer_links: {
+                    ancestor: [:index, { aliquots: %i[library tag tag2 aliquot_index sample] }]
```

Comment:
If you adjust the associations as suggested above, then don't forget to update this.